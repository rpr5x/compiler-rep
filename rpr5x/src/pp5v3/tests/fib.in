5
10
18
-1
