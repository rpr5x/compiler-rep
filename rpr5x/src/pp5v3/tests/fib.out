Loaded: /if14/ww6r/WorkingDisk/TA/compiler/exceptions/exceptions.s

This program computes Fibonacci numbers (slowly.. but correctly!)

Enter the fibonacci number you want: (-1 to quit) Fib(5) = 5

Enter the fibonacci number you want: (-1 to quit) Fib(10) = 55

Enter the fibonacci number you want: (-1 to quit) Fib(18) = 2584

Enter the fibonacci number you want: (-1 to quit) Goodbye!
